tests-extended/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+project(ExtendedTests)
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/../cmake/test_base.cmake)
+
+find_package(Boost REQUIRED system)
```

Comment:
No COMPONENTS keyword?

Reply:
By surprise, `it just works`. Kinda missed this one.